doc/releases.rst

```diff
@@ -9,6 +9,41 @@
 Releases
 ========
 
+Version 0.14.4
+==============
+
+* Fix #1356: SVR degree hyperparameter now only active with "poly" kernel.
+* Add #1311: Black format checking (non-strict).
+* Maint #1306: Run history is now saved every iteration incase of crashes.
+* Doc #1309: Updated doc faqs and manual
+* Doc #1322: Fix typo in contribution guide
+* Maint #1326: Add isort checker (non-strict)
+* Maint #1238, #1346, #1368, #1370: Update warnings in tests
+* Maint #1325: Test workflow can now be manually triggered
+* Maint #1332: Update docstring and typing of ``include`` and ``exclude`` params
+* Maint #1260: Add Python 3.10 to our tests
+* Add #1318: First update to use ``automl_common``
```

Comment:
Could add a few more details on what is exactly used from automl_common, explain that this is used via a submodule, and provide a link to the external repo?

Reply:
Will do